model/src/main/scala/ch.epfl.scala.index.model/release/SemanticVersion.scala

```diff
@@ -1,79 +1,82 @@
 package ch.epfl.scala.index.model.release
 
 /**
- * Semantic version, separation of possible combinations
- * @param major the major version number
- * @param minor the minor version number
- * @param patch the path version number
- * @param preRelease the pre release name
- * @param metadata the release metadata
- */
+  * Semantic version, separation of possible combinations
+  * @param major the major version number
+  * @param minor the minor version number
+  * @param patch the path version number
+  * @param preRelease the pre release name
+  * @param metadata the release metadata
+  */
 case class SemanticVersion(
-  major: Long,
-  minor: Long = 0,
-  patch: Option[Long] = None,
-  preRelease: Option[PreRelease] = None,
-  metadata: Option[String] = None
+    major: Long,
+    minor: Long = 0,
+    patch: Option[Long] = None,
+    preRelease: Option[PreRelease] = None,
+    metadata: Option[String] = None
 ) {
 
   /**
-   * display correct and nice version as string eg: 1.0.0 | 1.2.1 | 1.2 | 1.2.0-RC1
-   * @return
-   */
+    * display correct and nice version as string eg: 1.0.0 | 1.2.1 | 1.2 | 1.2.0-RC1
+    * @return
+    */
   override def toString = {
 
     val patchPart = patch.map("." + _).getOrElse("")
 
-    val preReleasePart = preRelease.map{
+    val preReleasePart = preRelease.map {
 
       case Milestone(d) => "M" + d.toString
       case ReleaseCandidate(d) => "RC" + d.toString
       case OtherPreRelease(v) => v.toString
     }.map("-" + _).getOrElse("")
 
     val metadataPart = metadata.map("+" + _).getOrElse("")
-    
+
     major + "." + minor + patchPart + preReleasePart + metadataPart
   }
 }
 
 object SemanticVersion {
 
   /**
-   * special ordering for versions
-   * @return
-   */
+    * special ordering for versions
+    * @return
+    */
   implicit def ordering = new Ordering[SemanticVersion] {
     val LT = -1
-    val GT =  1
-    val EQ =  0
-    
+    val GT = 1
+    val EQ = 0
+
     val lcmp = implicitly[Ordering[Long]]
     val scmp = implicitly[Ordering[String]]
     val cmp = implicitly[Ordering[(Long, Long, Option[Long])]]
 
-    def compare(v1: SemanticVersion, v2: SemanticVersion): Int = {     
+    def compare(v1: SemanticVersion, v2: SemanticVersion): Int = {
       def tupled(v: SemanticVersion) = {
         import v._
         (major, minor, patch)
       }
-      
+
       val tv1 = tupled(v1)
       val tv2 = tupled(v2)
-      
+
       def preCmp(pr1: Option[PreRelease], pr2: Option[PreRelease]): Int = {
         (pr1, pr2) match {
-          case (None, None)                                               => EQ
```

Comment:
Yes, if you enable vertical alignment. I'm not a huge fan, personally, I can do it if you're so interested in it.